src/processors/full.ts

```diff
@@ -27,6 +27,8 @@ export class Full extends BaseProcessor {
   
   init() {
     this.logger.info("Initialising XDC relayer");
+    //TODO: check CSC mode is correct at init
```

Comment:
could u add an github issue and link it here?

Reply:
changed my mind on this, doesn't look as easy as I thought
also, we don't really need this at this stage